kernelci/build.py

```diff
@@ -633,11 +633,12 @@ def build_kernel(build_env, kdir, arch, defconfig=None, jopt=None,
             result = _run_make(target='dtbs', **kwargs)
     build_time = time.time() - start_time
 
-    if result and mods and mod_path:
-        abs_mod_path = os.path.join(output_path, mod_path)
-        if os.path.exists(abs_mod_path):
-            shutil.rmtree(abs_mod_path)
-        os.makedirs(abs_mod_path)
+    if result and mods:
+        if not mod_path:
```

Comment:
It's better to do this at the top of the function where the other arguments are assigned default values such as `output_path`.

Reply:
Also the docstring should be updated to explain the default value of `mod_path`, see below for a similar case with `install_path`.